common/src/main/webapp/sass/explore.scss

```diff
@@ -3011,6 +3011,19 @@ div.spectroscopy-table-emptymessage {
     padding: 0.3em;
   }
 
+  .explore-proposal-submission-line {
+    display: flex;
+    align-items: center;
+  }
+
+  .explore-proposal-deadline {
+    .p-inline-message-text {
+      /* stylelint-disable-next-line csstools/value-no-unknown-custom-properties */
```

Comment:
Why is this directive necessary?

Reply:
because of the variable, at least I got an error with stylelint locally